.gitattributes

```diff
@@ -0,0 +1,7 @@
+/examples export-ignore
```

Comment:
Did I miss this earlier or did you just add `/examples` to the list as well ?

Examples should probably be considered documentation, similar to the `README` and would, in that case, normally be included in a distribution.

Reply:
No it was already there

Reply:
:grin: In that case, I haven't said anything...